docs/flipflop/flipflops.md

```diff
@@ -15,6 +15,18 @@ In first method, cascade two latches in such a way that the first latch is enabl
 
 In second module, we can directly implement the flip-flop, which is edge sensitive. In this module, let us discuss the following flip-flops using second method.
 
+## Difference between Latch and Flip Flop
+
+
+| Latch       | Flip flop     |
+|:------------|:--------------|
+| Latches are level triggered | Flip flops are edge triggered |
```

Comment:
No need for `Latches are` and `Flip flops are`

You can just use `Level triggered | Edge triggered` as the table headings explain the rest. This also applies to the other points